communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "pypa.png",
+      "title": "PyPa",
+      "quote": "Some Python Packaging Authority projects (such as virtualenv) use Discord to answer questions from users and to collaborate on development tasks.",
+      "inviteCode": "gDvaHVJvsn",
```

Comment:
Is this an official server that you're just settings up. There is only 1 member in this community

![Screen Shot 2021-02-10 at 10 13 33 AM](https://user-images.githubusercontent.com/952783/107552653-a4d3cb80-6b88-11eb-9ab6-ef2b625b773c.png)

Reply:
Yeah, it's a new server. I'm the maintainer of the virtualenv package, for which I would like to move to discord. (and other projects might join in, some still prefer to use irc).